cmds/exp/gosh/gosh.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// Derived work from Daniel Martí <mvdan@mvdan.cc>
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	prompt "github.com/c-bata/go-prompt"
+	"github.com/c-bata/go-prompt/completer"
+	"golang.org/x/term"
+
+	"mvdan.cc/sh/v3/interp"
+	"mvdan.cc/sh/v3/syntax"
+)
+
+var tabCompletion = flag.Bool("tabcomp", false, "Enable tab completion.")
```

Comment:
false by default?

Reply:
Ah that was a leftover from testing, I think we can use tab completion by default so I'll change up the logic here.